### PR TITLE
Feature/heroletes marketplace

### PR DESCRIPTION
**Pull request by gastonri**
**Description**
We're adding Heroletes Marketplace, allowing the user to use p2p exchange for heroletes NFTs.
Also, we're allowing the user to sell the heroletes NFTs by reconditioning the nft by id page.

**Screenshots:**
image
image
image

### CODE REVIEW

File: src/app/interfaces/NFT/NFT.ts [hunk 1]
```diff
@@ -2,6 +2,8 @@ export interface NFT {
   id: string;
   name: string;
   ftx_id: string;
+  attributes: any;
```
Comment (Alejoboga20) [inline src/app/interfaces/NFT/NFT.ts:5]: can we try to type this?

File: src/app/pages/HeroletesMarketplace/HeroleteItem/HeroleteItem.tsx [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+  const onMouseEnter = (e: SyntheticEvent) => {
+    (e.currentTarget as HTMLVideoElement).play();
+  };
+
+  const onMouseLeave = (e: SyntheticEvent) => {
+    (e.currentTarget as HTMLVideoElement).pause();
+  };
+
+  return (
+    <Grid item xs={6} md={4} className={styles.heroleteItem}>
```
Comment (Alejoboga20) [inline src/app/pages/HeroletesMarketplace/HeroleteItem/HeroleteItem.tsx:28]: Lets move Grid item to HOC component to have grid displayed in same place

File: src/app/pages/HeroletesMarketplace/HeroletesList/HeroletesList.tsx [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+import { HeroleteItem } from '../HeroleteItem/HeroleteItem';
+import { useHeroleteList } from './useHeroleteList';
+
+export const HeroletesList = () => {
+  const { heroletes } = useHeroleteList();
+
+  return (
+    <Grid container>
+      {heroletes.map(herolete => (
+        <HeroleteItem
```
Comment (Alejoboga20) [inline src/app/pages/HeroletesMarketplace/HeroletesList/HeroletesList.tsx:11]: Lets move grid item here wrapping HeroleteItem it's much more clear

**Review by Alejoboga20 — APPROVED** [review]
Good jub, jsut a few minor comments.

File: src/app/interfaces/NFT/NFT.ts (hunk 1 above)
Comment (gastonri) [reply src/app/interfaces/NFT/NFT.ts:5]: We could do that for herolete, but the new tokens for hall of fame wont be the same and would lose point.

**Pull request merged**